iowa-a/bedrock-prod/canary-deploy.yaml

```diff
@@ -0,0 +1,253 @@
+        - name: FXA_OAUTH_CLIENT_ID
+          value: "40dd9c0253db092c"
+        - name: FXA_OAUTH_CLIENT_SECRET
+          valueFrom:
+            secretKeyRef:
+              key: fxa-oauth-client-secret
+              name: bedrock-prod-secrets
+        - name: GTM_CONTAINER_ID
+          value: GTM-MW3R8V
+        - name: GUNICORN_WORKER_CLASS
+          value: gevent
+        - name: HTTPS
+          value: "on"
+        - name: L10N_CRON
+          value: "True"
+        - name: LOGLEVEL
+          value: info
+        - name: MOFO_SECURITY_ADVISORIES_PATH
+          value: /tmp/mofo_security_advisories
+        - name: NEWRELIC_PYTHON_INI_FILE
+          value: newrelic.ini
+        - name: NEW_RELIC_APP_NAME
+          value: bedrock-prod-canary-iowa-a;bedrock-prod-canary
+        - name: NEW_RELIC_LICENSE_KEY
+          valueFrom:
+            secretKeyRef:
+              key: new-relic-license-key
+              name: bedrock-prod-secrets
+        - name: PROD
+          value: "True"
+        - name: SECURE_BROWSER_XSS_FILTER
+          value: "True"
+        - name: SECURE_HSTS_SECONDS
+          value: "31536000"
+        - name: SECURE_SSL_REDIRECT
+          value: "True"
+        - name: SWITCH_NEWSLETTER_MAINTENANCE_MODE
+          value: "False"
+        - name: SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              key: secret-key
+              name: bedrock-prod-secrets
+        - name: SENTRY_DSN
+          value: "https://dd94e99168c6431899c1daa5a812d8b6@sentry.prod.mozaws.net/152"
+        - name: STATIC_URL
+          value: "https://www.mozilla.org/media/"
+        - name: STATSD_CLIENT
+          value: django_statsd.clients.normal
+        - name: STD_SMS_COUNTRIES_DESKTOP
+          value: US,DE,FR
+        - name: STD_SMS_COUNTRIES_WHATSNEW50
+          value: US,DE,FR
+        - name: STD_SMS_COUNTRIES_WHATSNEW61
+          value: US,DE
+        - name: STUB_ATTRIBUTION_HMAC_KEY
+          valueFrom:
+            secretKeyRef:
+              key: stub-attribution-hmac-key
+              name: bedrock-prod-secrets
+        - name: STUB_ATTRIBUTION_RATE
+          value: '1'
+        - name: SWITCH_TRACKING_PIXEL
+          value: 'on'
+        - name: TWITTER_ACCESS_TOKEN
+          valueFrom:
+            secretKeyRef:
+              key: twitter-access-token
+              name: bedrock-prod-secrets
+        - name: TWITTER_ACCESS_TOKEN_SECRET
+          valueFrom:
+            secretKeyRef:
+              key: twitter-access-token-secret
+              name: bedrock-prod-secrets
+        - name: TWITTER_CONSUMER_KEY
+          valueFrom:
+            secretKeyRef:
+              key: twitter-consumer-key
+              name: bedrock-prod-secrets
+        - name: TWITTER_CONSUMER_SECRET
+          valueFrom:
+            secretKeyRef:
+              key: twitter-consumer-secret
+              name: bedrock-prod-secrets
+        - name: WEB_CONCURRENCY
+          value: "6"
```

Comment:
This means we run 6 gevent workers per pod, which is the same number we did for meinheld?

Reply:
yes, same as meinheld.  Figured starting here with the canary made sense.